README.md

```diff
@@ -1 +1,11 @@
-``` shell
g++ -static lib_sample_use.cpp -std=c++11 -lfastFM -lglog -pthread -I. -L.
```
\ No newline at end of file
+Build:
+
+    > $GCC_MUSL_ROOT/x86_64-linux-musl-g++ -static -std=c++11 lib_sample_use.cpp libfastFM.a libglog.a -o foo
```

Comment:
I'm not sure if we can use regular g++. Probably that was the reason of the original problem. Since static build doesn't seem to work my only suggestion is to try shared library. If it will not work too, then I can guess it's not possible in general.